src/tsickle.ts

```diff
@@ -126,7 +139,7 @@ class ClosureRewriter extends Rewriter {
         let newTag: jsdoc.Tag = {
           tagName: 'param',
           optional: paramNode.initializer !== undefined || paramNode.questionToken !== undefined,
-          parameterName: unescapeName(paramSym.getName()),
+          parameterName: makeLegalExternsParam(unescapeName(paramSym.getName())),
         };
```

Comment:
It's not a bad idea.  So far we haven't been putting in human-readable comments into the JS (except in cases where it helps read the results of the test suite) because we don't intend for humans to use the resulting output.  I will file a bug on this idea.